Football/src/navigation/StackNavigator/index.tsx

```diff
@@ -1,25 +1,38 @@
-import React from 'react';
-import {useSelector} from 'react-redux';
-import {createStackNavigator} from '@react-navigation/stack';
-import {BottomTabNavigator} from '../BottomTabNavigator';
-import {AuthenticationScreen} from '../../screens/AuthenticationScreen';
+import React, { FC } from 'react';
+import { useSelector } from 'react-redux';
+import { createStackNavigator } from '@react-navigation/stack';
+import { BottomTabNavigator } from 'navigation/BottomTabNavigator';
+import { AuthScreen } from 'screens/AuthScreen';
+import { RootState } from 'reducers/types';
 
 const Stack = createStackNavigator();
 
-export const StackNavigator = () => {
-  const userToken = useSelector((state) => state['authReducer'].userToken);
+export const StackNavigator: FC = () => {
+  const userToken = useSelector((state: RootState) => state.authReducer.userToken);
+
   return (
-    <Stack.Navigator>
-      {userToken == null ? (
+    <Stack.Navigator
+      screenOptions={{
+        headerStyle: {
+          backgroundColor: '#277D38',
```

Comment:
how it looks like in dark/white color theme? Maybe need to add that color in themes and create a separate variable for COLORS